packages/joint-core/src/dia/Paper.mjs

```diff
@@ -1759,59 +1830,115 @@ export const Paper = View.extend({
         return restrictedArea;
     },
 
-    createViewForModel: function(cell) {
+    _resolveCellViewPlaceholder: function(placeholder) {
+        const { model, viewClass, cid } = placeholder;
+        const view = this._initializeCellView(viewClass, model, cid);
+        view.paper = this;
+        this._views[model.id] = view;
```

Comment:
This part can be converted into the separate function or just put into the `createViewForModel` method. There is very similar part in the `renderView` method (line 1972)

Reply:
Great point!